mmv1/third_party/terraform/acctest/bootstrap_test_utils.go.tmpl

```diff
@@ -1435,6 +1439,89 @@ func SetupProjectsAndGetAccessToken(org, billing, pid, service string, config *t
 	return accessToken, nil
 }
 
+{{ if ne $.TargetVersionName `ga` -}}
+	// BootstrapSharedCodeRepositoryIndex will create a code repository index
+	// if it hasn't been created in the test project.
+	//
+	// BootstrapSharedCodeRepositoryIndex returns a persistent code repository index
+	// for a test or set of tests.
+	//
+	// Deletion of code repository index takes a few minutes, and creation of it
+	// currently takes about half an hour.
+	// That is the reason to use the shared code repository indexes for test resources.
+	const SharedCodeRepositoryIndexPrefix = "tf-bootstrap-cri-"
+
+	func BootstrapSharedCodeRepositoryIndex(t *testing.T, codeRepositoryIndexId, location, kmsKey string, labels map[string]string) string {
```

Comment:
Currently not used, it was created for another resource for which I will create a PR after that. I can remove it for now but it will certainly be used.